cleaning/big_clean.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# second script of lab 0
+##### CONSTANTS
+ARCHIVE_NAME=$1
+BASE=$(basename -s .tgz "$ARCHIVE_NAME")
```

Comment:
To someone not familiar with the lab, it's not obvious what `BASE` is or what it's used for, especially since you don't use it until near the end of the script. You could either comment it here, or move it closer to where it's used, or both?

Reply:
Nice. 💯